File: Practice(Beginner)/Python3/Malvika_is_peculiar_about_color_of_balloons.py
# Source: https://www.codechef.com/problems/CHN09

t = int(input())
for test in range(t):
    s = input()
    cnt_a = s.count('a')
    cnt_b = len(s) - cnt_a
    print(min(cnt_a,cnt_b))

File: Practice(Beginner)/C++/Malvika is peculiar about color of balloons.cpp
// Source: https://www.codechef.com/problems/CHN09

#include <bits/stdc++.h>
using namespace std;

int main() {
    int t;
    cin >> t;
    while (t--){
        string s;
        cin >> s;
        int cnt_a = count(s.begin(),s.end(),'a');
        int cnt_b = s.length() - cnt_a;
        cout << min(cnt_a,cnt_b) << endl;
    }
    return 0;
}